portfolio/src/main/webapp/script.js

```diff
@@ -48,10 +40,8 @@ function showSlides(n) {
   }
 
   for (i = 0; i < slides.length; i++) {
-      slides[i].style.display = "none";
-  }
-  for (i = 0; i < dots.length; i++) {
```

Comment:
Is it possible to use for-of? The style guide says:

With ES6, the language now has three different kinds of for loops. All may be used, though for-of loops should be preferred when possible.

Reply:
I read more into for-of loops and it is possible, however, I would need  to use two for loops one for the dots and one for the slides. In this case, would it be better to loop once using a transitional `for` loop or twice using `for-of`?

Reply:
I am not a js person, so I would say do what is easiest :)